starter-code/index.html

```diff
@@ -9,7 +9,7 @@
   <link rel="stylesheet" href="styles/icons.css">
   <link rel="stylesheet" href="styles/layout.css">
   <link rel="stylesheet" href="styles/modules.css">
-  <link rel="icon" type="image/x-icon" href="https://avatars2.githubusercontent.com/u/18607604?v=3&s=200">
+  <!--<link rel="icon" type="image/x-icon" href="https://avatars2.githubusercontent.com/u/18607604?v=3&s=200-->
```

Comment:
Not sure why this was commented out. If you don't need it for part of the assignment you should remove it before submitting =)